README.md

```diff
@@ -36,40 +33,36 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - run: composer install --ansi --no-progress --no-interaction --prefer-dist
-      - run: mkdir vendor-zip
       - uses: montudor/action-zip@v0.1.0
         with:
-          args: zip -qq -r ./vendor-zip/vendor.zip ./vendor
-      - uses: actions/upload-artifact@master
+          args: zip -qq -r vendor.zip vendor
+      - uses: actions/upload-artifact@v2
         with:
-          name: vendor
-          path: vendor-zip
+          name: vendor.zip
   tests:
     needs: composer-install
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/download-artifact@master
+      - uses: actions/download-artifact@v2
         with:
-          name: vendor
-          path: vendor-zip
+          name: vendor.zip
       - uses: montudor/action-zip@v0.1.0
         with:
-          args: unzip -qq ./vendor-zip/vendor.zip -d ./vendor
-      - run: rm -Rf ./vendor-zip
+          args: unzip -qq vendor.zip -d vendor
+      - run: rm vendor.zip
```

Comment:
Is there a reason why this is needed?

Reply:
I didn't add this example (see #2) but I don't believe this is a necessary step